src/assets/i18n/fr_FR.json

```diff
@@ -923,6 +938,13 @@
   "navigation.paging.previous_page.label": "Aller à la page précédente",
   "order.tracking.error": "Malheureusement, nous n’avons pas pu localiser une commande avec les informations que vous avez fournies.",
   "order_template.create.heading": "Créer un modèle de commande",
+  "organization.group.label": "Groupe :",
+  "organization.groups.description.label": "Description",
+  "organization.groups.name.label": "Nom",
+  "organization.groups.new.name.error.required": "Veuillez saisir une URL.",
+  "organization.groups.new.parent.error.required": "Veuillez sélectionner un groupe de parents.",
+  "organization.groups.parent.label": "Groupe parent",
+  "organization.groups.parent.placeholder": "Sélectionner un groupe parent",
```

Comment:
Please double-check if "parent" is a suitable translation here or whether it should rather be "groupe principal".

Reply:
Should be "groupe principal"